### DISCUSSION TITLE
How to implement scrollTo() function in react

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [idiotWu/smooth-scrollbar#375] [Q&A]
I have a function to call scrollTo (), but that doesn't work, and I get an error, TypeError: scrollbar.scrollTo is not a function
sample code :
const scrollToTop = () => {
    scrollbar.scrollTo(0, 100, 600);
  };

**Comment by sadeghbarati** [comment]
Are you using [react-smooth-scrollbar](https://www.npmjs.com/package/react-smooth-scrollbar)?

this package is recommended if you using `react`

use react `Context` to getScrollbar then use smoothScrollBar api

**Reply by mhanifmuhsin** [reply]
Do you mean context it is useContext?

**Reply by sadeghbarati** [reply]
take a look at the npm package link...

https://www.npmjs.com/package/react-smooth-scrollbar#get-scrollbar-instance

**Reply by dolphin-wood** [reply]
I have not been using react for years... Is that possible to use this plugin directly in react projects since [react-smooth-scrollbar](https://github.com/idiotWu/react-smooth-scrollbar) is no longer maintained?

**Reply by mhanifmuhsin** [reply]
I am also the same, by the way thank for your answer, and I hope that in the future it can be developed again, because it will be very useful
